### PR TITLE
feat: configurable readme path

### LINKED ISSUE #1218: Make README.md filename configurable
I wish the name of the file was configurable because it conflicts with another tool that generates a README already. And I wish to store the codegen'ed README in another file, e.g. CLI.md and then either link to it, or include it in the main `README.md`

https://github.com/oclif/oclif/blob/3570d93f5eac061a6b59518bf69a73fb013e1754/src/commands/readme.ts#L175

I am willing to provide a PR.

### PR DESCRIPTION
**Pull request by moltar**
Closes #1218

### CODE REVIEW

**Comment by moltar** [comment]
> Could not sign the CLA, please contact: oss-bots@salesforce.com

**Comment by moltar** [comment]
screenshot-20231214T210759-HESlnGHq@2x

File: src/commands/readme.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
     aliases: Flags.boolean({allowNo: true, default: true, description: 'include aliases in the command list'}),
     dir: Flags.string({default: 'docs', description: 'output directory for multi docs', required: true}),
     multi: Flags.boolean({description: 'create a different markdown page for each topic'}),
+    readmePath: Flags.string({default: 'README.md', description: 'Path to the README file.', required: true}),
```
Comment (mdonnalley) [inline src/commands/readme.ts:35]: can you rename this to `readme-path` to stay consistent with our naming convention? Other than that, this change looks great!

**Comment by moltar** [comment]
screenshot-20231214T210840-SnOTws3E@2x

🤷🏼

**Review by mdonnalley — CHANGES_REQUESTED** [review]

**Review by mdonnalley — APPROVED** [review]

**Comment by mdonnalley** [comment]
Thanks @moltar! I merged this into a separate branch so that we can run all the integration tests before merging to main. I'll merge this as soon as all the tests pass

**Pull request merged**